### ISSUE TITLE
Refactor Popcorn Time to source the CE version from Github

### ISSUE THREAD

**Issue by guinuxbr** [folkswithhats/fedy#452]
According to this: http://popcorn-time.is/official-statement.html#Malware2, the PopcornTime installer that is available at popcorntime.sh is a threat. Maybe the URL should be replaced by the PopcornTime CE.
This is the actual Popcorn Time URLs.
http://popcorn-time.is/
https://github.com/PopcornTimeCommunity/desktop

**Comment by bddenhartog** [comment]
I'm considering this a **high priority** and will resolve it shortly.

**Comment by bddenhartog** [comment]
Patch submitted for review, but before we merge this in, I have a few questions.

- Why is `PopcornTimeCommunity` more trustworthy than `popcorn-official`? Both applications are open-sourced:
    - https://github.com/PopcornTimeCommunity/desktop
    - https://github.com/popcorn-official/popcorn-desktop
- [This image](https://i.imgur.com/cRBrr6z.png) from the subreddit [/r/PopCornTime](https://reddit.com/r/PopCornTime) seems to directly contradict the image displayed in the blog post you linked. What gives?

**Comment by bddenhartog** [comment]
Additionally:

* I ran [a scan](http://archive.is/lLSZQ) with `virustotal.com` with the download provided by `popcorntime.sh` for Windows 7+. There are no scanners that reported the file as unsafe, including Symantec (the only scanner that reported 'unsafe' on the file in the scan linked in the article).

* The file name in the blog post's `virustotal.com` scan is `Popcorn-Time Official 2016-Setup-ByINFORMACIONLIBRE.exe`, which is not the file name of the download provided by `popcorntime.sh`.

In other words, I can't tell that the information in the article you linked is accurate. It looks clean by all accounts.

**Comment by guinuxbr** [comment]
Hi... After a little more research I can assume that you are right. There's no easy way to determine which domain is real or fake. They all claim to be clean. I'm sorry for this.
